AdobeFlashPlayer/AdobeFlashURLProvider.py

```diff
@@ -34,15 +35,25 @@ class AdobeFlashURLProvider(Processor):
     """Provides URL to the latest Adobe Flash Player release."""
     description = __doc__
     input_variables = {
-        "url": {
-            "required": False,
+        'url': {
+            'required': False,
             "description": ("Override URL. If provided, this processor "
                             "just returns without doing anything."),
         },
-        "version": {
+        'request_headers': {
+            'description': ('Optional dictionary of headers to include with '
+                            'the download request.'),
+            'required': False,
+        },
+        'curl_opts': {
+            'description': ('Optional array of curl options to include with '
+                            'the download request.'),
+            'required': False,
+        },
+        "CURL_PATH": {
             "required": False,
-            "description": ("Specific version to download. If not defined, "
-                            "defaults to latest version.")
+            "default": "/usr/bin/curl",
+            "description": "Path to curl binary. Defaults to /usr/bin/curl.",
         },
     }
```

Comment:
Why did you remove the "version" input variable? This will break people who use it.

Reply:
This is an oversight. I will fix it.